scripts/schemas/board-schema.yml

```
@@ -99,6 +99,11 @@ mapping:
   runners:
     type: map
     mapping:
+      priority:
+        type: int
+        desc: |
+              Priority of this flash run once configuration from 0 to 100. The highest value data
```

The 0--100 range is not enforced, but I guess it doesn't need to be. It may as well be an arbitrarily large positive or negative integer.

```suggestion
              Priority of this flash run once configuration. The highest value data
```

---

Fixed

---

What changed?

---

ah woops did a "git status ." in the wrong directory, will add files